cfn/paws-collector.template

```diff
@@ -388,6 +450,16 @@
                         "Arn"
                     ]
                 },
+                "AuthType": {
+                    "Ref": "PawsAuthType"
+                },
+                "CredsBucket": {
+                    "Fn:If": [
+                        "CreateCredentialsBucket",
+                        {"Fn::Sub" : ["${ImReallySorryIDidThisHack}", {"ImReallySorryIDidThisHack": {"Fn::GetAtt" : ["CredentialsBucket", "BucketName"]}}]},
```

Comment:
Why can't we have
```
"CredsBucket": {
                    "Ref": "PawsCredsBucket"
                },
```

Reply:
My understandingis that the If function needs a string. this is a hack to turn that bucket name into a string. Though i think that is going to need some testing. the mysteries of cloud formation...